lib/suspenders/generators/app_generator.rb

```diff
@@ -233,6 +238,14 @@ def remove_routes_comment_lines
       build :remove_routes_comment_lines
     end
 
+    def generate_all
+      run("spring stop")
```

Comment:
It looks like people chiming into suspenders are typically against `spring`, yet we never managed to drop it.